### ISSUE TITLE
feature request: please add https://toonlens.com/ model as option

### ISSUE THREAD

**Issue by cocoonkid** [divamgupta/diffusionbee-stable-diffusion-ui#119]
feature request: please add https://toonlens.com/ model as option

That would be really great!

**Comment by camilovietnam** [comment]
Is this model even available publicly? I see that they charge $0.99 per picture, how would that be integrated into Diffusion Bee?

**Comment by cocoonkid** [comment]
@camilovietnam Yes I understand not adding a paid thing of course. The price ask is pretty recent.

I will try identifying the model that is in use. I'm pretty sure something similar is out there for free!
